### PR TITLE
WOR-1510 update tcl, otel and better http metrics buckets

### PR DESCRIPTION
**Pull request by dvoet**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-1510
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

**Review by aherbst-broad — APPROVED** [review]

File: core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/TracingDirectives.scala [hunk 1]
```diff
@@ -3,12 +3,18 @@ package org.broadinstitute.dsde.rawls.metrics
 import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
 import akka.http.scaladsl.server.Directives.{extractRequest, handleExceptions, mapResponse, provide}
 import akka.http.scaladsl.server.{Directive1, ExceptionHandler}
+import bio.terra.common.opentelemetry.HttpServerMetrics
```
Comment (dvoet) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/TracingDirectives.scala:6]: this version of `HttpServerMetrics` has the http metrics buckets we want

File: project/Dependencies.scala [hunk 2]
```diff
@@ -128,27 +128,17 @@ object Dependencies {
   val dataRepo = clientLibExclusions("bio.terra" % "datarepo-jakarta-client" % "1.568.0-SNAPSHOT")
   val resourceBufferService = clientLibExclusions("bio.terra" % "terra-resource-buffer-client" % "0.198.42-SNAPSHOT")
   val billingProfileManager = clientLibExclusions("bio.terra" % "billing-profile-manager-client" % "0.1.506-SNAPSHOT")
-  val terraCommonLib = tclExclusions(clientLibExclusions("bio.terra" % "terra-common-lib" % "0.1.9-SNAPSHOT" classifier "plain"))
+  val terraCommonLib = tclExclusions(clientLibExclusions("bio.terra" % "terra-common-lib" % "0.1.11-SNAPSHOT" classifier "plain"))
   val sam: ModuleID = clientLibExclusions("org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-70fda75")
   val leonardo: ModuleID = "org.broadinstitute.dsde.workbench" % "leonardo-client_2.13" % "1.3.6-d0bf371"
 
   // OpenTelemetry
-  val openTelemetryVersion = "1.31.0"
```
Comment (dvoet) [inline project/Dependencies.scala:136]: all of the removed otel dependencies are inherited from TCL

**Comment by dvoet** [comment]
jenkins retest

**Comment by dvoet** [comment]
jenkins retest

**Comment by dvoet** [comment]
jenkins retest

**Review by cahrens — APPROVED** [review]

**Pull request merged**